ci-operator/step-registry/openshift/e2e/test/openshift-e2e-test-commands.sh

```diff
@@ -285,6 +281,7 @@ do
   echo "$(date) - $MACHINECOUNT Machines - $NODECOUNT Nodes"
   sleep 30
   ((i++))
+  # TODO Help me bash experts!  I want to wait for up to 20 minutes on Azure and keep this at 10 minutes for everything else.
```

Comment:
@JoelSpeed does the run timeline support choosing 20 minutes after installation completion on Azure?

Reply:
Based on our thread slack and the analysis we looked at earlier, I don't think extending this is actually going to help. Going from machine object created to ignition requested seems to only be in the order of a few minutes. Add to that the approx 3 mins for ignition to kubelet starting fully, I think 10 minutes should be adequate when we factor in the existing delays the installer process has